Example/Kommunicate_ExampleUITests/KommunicateLoginAsVisitorUITests.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  KommunicateLoginAsVisitorUITests.swift
+//  Kommunicate_ExampleUITests
+//
+//  Created by Kommunicate on 02/07/20.
+//  Copyright © 2020 CocoaPods. All rights reserved.
+//
+
+import XCTest
+
+class KommunicateLoginAsVisitorUITests: XCTestCase {
+    enum GroupData {
+        static let typeText = "verifying Login as a visitor and FAQ button."
+        static let AppId = "TestAppId"
+    }
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        addUIInterruptionMonitor(withDescription: AppPermission.AlertMessage.accessNotificationInApplication) { (alerts) -> Bool in
+            if alerts.buttons[AppPermission.AlertButton.allow].exists {
+                alerts.buttons[AppPermission.AlertButton.allow].tap()
+            }
+            return true
+        }
+        let app = XCUIApplication()
+        if let appId = appIdFromEnvVars() {
+            app.launchArguments = ["-appId", appId]
+        }
+        app.launch()
+        sleep(5)
```

Comment:
Decrease the value or remove it if not required.

Reply:
It is required